go.mod

```diff
@@ -3,13 +3,15 @@ module github.com/crossplane/provider-template
 go 1.13
 
 require (
-	github.com/crossplane/crossplane-runtime v0.10.0
+	github.com/crossplane/crossplane v0.14.0 // indirect
+	github.com/crossplane/crossplane-runtime v0.12.0
 	github.com/crossplane/crossplane-tools v0.0.0-20201007233256-88b291e145bb
-	github.com/google/go-cmp v0.4.0
+	github.com/crossplane/provider-aws v0.15.0 // indirect
+	github.com/google/go-cmp v0.5.0
```

Comment:
I believe at least `crossplane/crossplane` and `crossplane/provider-aws` should not be present here.

Reply:
Made the mistake of running `gopls` while trying to edit these files. :( Fixed.